### ISSUE TITLE
Cannot allocate memory

### ISSUE THREAD

**Issue by zxshi** [tugstugi/pytorch-saltnet#3]
I use `python train.py --vtf --pretrained imagenet --loss-on-center --batch-size 6 --optim adamw --learning-rate 5e-4 --lr-scheduler noam --basenet resnet34 --max-epochs 100 --data-fold fold0 --log-dir runs/fold0 --resume runs/fold0/checkpoints/last-checkpoint-fold0.pth
` to train my dataset. My train data have 71712 images and valid data have 7968 images. I got this error:
<img width="515" alt="0f69702d7a503b0aeb6e055814831c9" src="https://user-images.githubusercontent.com/13671645/51659708-63dedd80-1fe6-11e9-9d70-d5cdde952411.png">

**Comment by xuyuan** [comment]
The dataset tries to preload all image to RAM, you can try to disable preload in line:
https://github.com/tugstugi/pytorch-saltnet/blob/a3e63b357f975924e75b5db24ee528c5797f5efb/train.py#L107
and
https://github.com/tugstugi/pytorch-saltnet/blob/a3e63b357f975924e75b5db24ee528c5797f5efb/train.py#L109
